packages/react-native-renderer/src/ReactNativeTypes.js

```diff
@@ -89,7 +89,7 @@ class ReactNativeComponent<Props> extends React.Component<Props> {
   measure(callback: MeasureOnSuccessCallback): void {}
   measureInWindow(callback: MeasureInWindowOnSuccessCallback): void {}
   measureLayout(
-    relativeToNativeNode: number,
+    relativeToNativeNode: number | Object,
```

Comment:
Let's make this `number | ReactNativeComponent` and cast it on the inside. It's not correct but more specific.